libmscore/stafftype.cpp

```diff
@@ -240,6 +242,7 @@ void StaffType::write(XmlWriter& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering",       _showTabFingering);
```

Comment:
... can I do that online, a I have already deleted may branch after it had been merged ...?